snowplow_tracker/test/unit/test_tracker.py

```diff
@@ -53,54 +47,61 @@
 # helpers
 _TEST_UUID = '5628c4c6-3f8a-43f8-a09f-6ff68f68dfb6'
 geoSchema = "iglu:com.snowplowanalytics.snowplow/geolocation_context/jsonschema/1-0-0"
-geoData = {"latitude": -23.2,"longitude": 43.0}
+geoData = {"latitude": -23.2, "longitude": 43.0}
 movSchema = "iglu:com.acme_company/movie_poster/jsonschema/2-1-1"
 movData = {"movie": "TestMovie", "year": 2021}
 
-def mocked_uuid():
+
+def mocked_uuid() -> str:
     return _TEST_UUID
 
-def mocked_track(pb):
+
+def mocked_track(pb: Any) -> None:
     pass
 
-def mocked_complete_payload(*args, **kwargs):
+
+def mocked_complete_payload(*args: Any, **kwargs: Any) -> None:
     pass
 
-def mocked_track_trans_item(*args, **kwargs):
+
+def mocked_track_trans_item(*args: Any, **kwargs: Any) -> None:
     pass
 
-def mocked_track_unstruct(*args, **kwargs):
+
+def mocked_track_unstruct(*args: Any, **kwargs: Any) -> None:
     pass
 
+
 class ContractsDisabled(object):
-    def __enter__(self):
-        disable_all()
-    def __exit__(self, type, value, traceback):
-        enable_all()
+    def __enter__(self) -> None:
+        disable_contracts
```

Comment:
This is a really good catch! It actually means that the `ContractsDisabled` was useless since all the tests passed without it doing anything. I added the function call and removed `ContractsDisabled` from it where it wasn't necessary anymore. Also added a couple of tests that make use of it.